src/components/TodoList.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+// import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+interface Props {
+  todos: Todo[];
+  deleteTodo: (todoId: number) => void;
+}
+
+export const TodoList: React.FC<Props> = ({ todos, deleteTodo }) => {
+  return (
```

Comment:
if don't have any logic then delete the word `return` and return jsx in round brackets

`}) => (`

p.s. of course if linter allows

if you can do that, please check and update in other components as well